src/PKSim.Infrastructure/ProjectConverter/v11/Converter10to11.cs

```diff
@@ -152,5 +153,68 @@ private void addExpressionProfilesUsedBySimulationSubjectToProject(ISimulationSu
 
          _converted = true;
       }
+
+      private bool isChangedByCreatedIndividual(IParameter parameter)
```

Comment:
moved it here for now... I am going to use it do convert a project to v11.
ALternatively, we could read the info from the db but I do not know how I can retreive the ParameterInfo from DB for a given parmaeter instance? I am sure we have a way somewhere. Do you know @Yuri05 ?